source/reporting/Report database.rst

```diff
@@ -0,0 +1,248 @@
+ReportingService: Documentation of tables
```

Comment:
I think we should avoid the use of "ReportingService" and instead talk about it as the Reporting SQL Database (or just Reporting Database).

No point in giving external parties a headache about the fact its wrapped in its own service :)

Reply:
Updated!